CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -376,6 +442,11 @@ public void RegisterJsObject(string name, object objectToBind, BindingOptions op
                                     "called before the underlying CEF browser is created.");
             }
 
+            if (!initialized)
```

Comment:
Littering the code with this check and method call is far from ideal. I do understand the reason for delayed creation of unmanaged objects. If it has to be, then let's more the check into the method it's self.